docs/changes.txt

```diff
@@ -483,6 +483,8 @@ All (GUI):
 - Allow suppressing warnings from wxImage::LoadFile().
 - Allow customizing wxRibbon highlight colours (wxBen).
 - Enable selecting opacity in generic wxColourPicker.
+- Extend the support for font weights to a bigger range
```

Comment:
You're going to hate me for nitpicking, but could you please add full stops at the end of the new lines here for consistency with all the others?

Other than that, I don't see anything, let's merge this!

Reply:
No, I really appreciate your comments. It helps getting better results, and I'll try to remember :-)